lib/interface_quda.cpp

```diff
@@ -74,8 +74,7 @@
 
 #include <momentum.h>
 
-
-#include <cuda_profiler_api.h>
+#include <quda_profiler_api.h>
```

Comment:
This is the name of an actual CUDA include file, and its name should not be changed.  Though I'm not sure if this is even needed anymore, I would suggest to try deleting it.